### DISCUSSION TITLE
[Feature]: Flutter flavors

### DISCUSSION THREAD

**Discussion by Mortein** [immich-app/immich#1635] [Feature Request]
### Feature detail

Add a Development/Debug flavor to the Flutter app. Appends the non-production name to the app id (`app.alextran.immich` would become `app.alextran.immich-dev`). https://docs.flutter.dev/deployment/flavors

Allows more than one version of the app to be installed at the same time, such as the production version and a development version, as they no longer share the same application ID. I know this works well on Android, not sure about iOS sorry.

Not sure if there's a Test/Beta version as well, but that could be a third flavor.

To make it easier to tell them apart, use a different icon for each. Maybe the standard Immich logo, but with progressively fewer coloured petals from each layer (Production->Test->Development/Debug).

### Platform

Mobile App

**Comment by alextran1502** [comment]
Love this one!
